### PR TITLE
Add YouTube information

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Also make this the centre bad item on the front page for a bit of promotion

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
> I think we should have some technical documentation going with it. At least who's the owner of the channel etc in case people want to add things. What kind of things would we add, etc?

Yes, I think that's a good idea. I can add the technical information (it's me who "owns" it, for now) and have a first pass at some policy.

**Comment by graeme-a-stewart** [comment]
It seems that I can also make other people *owners* of the channel (so able to manage things like adding new managers, presumably). It would make a lot of sense to have at least 2 owners.

**Comment by graeme-a-stewart** [comment]
Let me just make another little note about the obscure URLs. YouTube does offer more memorable custom URLs, but you have to fulfil a few criteria:

```
Custom URL eligibility

To create a custom URL for your channel, your account needs to:

    Have 100 or more subscribers.
    Be at least 30 days old.
    Have an uploaded photo as a channel icon.
    Have uploaded channel art.
```

From <https://support.google.com/youtube/answer/2657968?hl=en>. So we need to wait a bit for that.

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - did that. Would you and/or Benedikt be willing to co-own the channel with me? I think it's more for backup than any need to do a lot.

**Comment by eduardo-rodrigues** [comment]
> Thanks @eduardo-rodrigues - did that. Would you and/or Benedikt be willing to co-own the channel with me? I think it's more for backup than any need to do a lot.

Sure. Agreed that we need several people. Thanks.

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - added you and also upgraded you to a channel owner.

Once @hegner reviews the guide this should be good to merge.

**Comment by hegner** [comment]
And if you want to add me as owner - OK with me.

File: _layouts/main.html [hunk 1]
```diff
@@ -58,23 +58,15 @@ <h2>Meetings</h2>
...
-    workload management and accounting. The goal of the workshop is
-    to exchange best practice and discuss areas of future collaboration and R&D.
-  </p>
-  <p>We are organising this workshop to 
-    <a href="https://indico.cern.ch/event/908146/page/20187-virtual-meeting-guide">optimise
-    for an online event</a>, please 
-    <a href="https://indico.cern.ch/event/908146/registrations/">register</a> so we can
-    better plan the event.
+  <img href="https://www.youtube.com/channel/UCv4hukXGkCYvBClQMKzypnQ" src="/images/others/youtubeIcon.png"
+    alt="YouTube" style="display: block; width:70%; margin-left: auto; margin-right: auto;">
```
Comment (hegner) [inline _layouts/main.html:66]: it turns out that 70 % is waaaaay too huge to look nice

**Pull request merged**